tests/sles4sap/hana_install.pm

```diff
@@ -127,6 +128,12 @@ sub run {
                 assert_script_run "parted --script $device --wipesignatures -- mklabel gpt mkpart primary 1 -1";
                 $device .= is_multipath() ? '-part1' : '1';
             }
+            # Remove traces of LVM structures from previous tests before configuring
+            script_run 'lvremove -f vg_hana';
```

Comment:
Maybe `vg_hana` could be added in a variable, in that case it will be easy to change his name if needed, WDYT?

Reply:
Done. I added only a regression verification run in x86_64 on qemu for the changes: https://openqa.suse.de/tests/4944446